test/e2e/config_test.go

```diff
@@ -118,3 +124,960 @@ func assertOperatorCondition(t *testing.T, conditionType configv1.ClusterStatusC
 		t.Fatal(err)
 	}
 }
+
+func TestClusterMonitorPrometheusOperatorConfig(t *testing.T) {
+	const (
+		component     = "prom-operator"
```

Comment:
Do we actually use this abbreviation anywhere in managed objects?

Reply:
No, I'm just using it to add some context to the log when the test fails, but we could likely remove these and let the test name imply what has failed

Reply:
If so, could you change this and other abbreviations to full names?

Reply:
Maybe we can remove it entirely, and just pass in the statefulset or deployment name instead?